browser/ui/views/frame/brave_browser_view_layout.h

```diff
@@ -35,6 +35,15 @@ class BraveBrowserViewLayout : public BrowserViewLayout {
     sidebar_separator_ = sidebar_separator;
   }
 
+  void set_contents_container(views::View* container) {
+    const_cast<std::remove_const_t<decltype(contents_container_)>*>(
```

Comment:
We should not be messing with const here. We need a wrapper that can switch between regular and split view

Reply:
wait, what's going on here actually? Why does this need `remove_const_t` in the first place?